src/utils/focusTrapComponent.ts

```diff
@@ -42,12 +42,15 @@ export function connectFocusTrap(component: FocusTrapComponent): void {
     return;
   }
 
-  if (focusTrapEl.tabIndex == null) {
-    focusTrapEl.tabIndex = -1;
+  if (focusTrapEl.tabIndex === null) {
```

Comment:
This `if` block won't be executed as `tabIndex` is a number and won't ever be null.

Reply:
When no `tabIndex` is defined, the value will be `-1` if it's a non-focusable element by default. Otherwise, it will be `0`.